code/game/verbs/ooc.dm

```diff
@@ -3,6 +3,7 @@
 	set category = "OOC"
 
 	sanitize_and_communicate(/decl/communication_channel/ooc, src, message)
+	mob.log_individual_message(sanitize(message), INDIVIDUAL_OOC_LOG, key_name(src, include_name = FALSE))
```

Comment:
Communication channel logging is handled by setting https://github.com/NebulaSS13/Nebula/blob/dev/code/datums/communication/channel.dm#L8 approparitely

Reply:
Don't really get how this work